examples/complete/variables.tf

```diff
@@ -25,5 +31,5 @@ variable "resource_group" {
 variable "resource_tags" {
   type        = list(string)
   description = "Optional list of tags to be added to created resources"
-  default     = []
+  default     = ["websphere", "liberty", "operator"]
```

Comment:
Don't set value here - the test will pass tags in

Reply:
coming from a test, removed